### PR TITLE
IIO node name changes

### PR DESCRIPTION
**Pull request by glima**

### CODE REVIEW

**Comment by bdilly** [comment]
Makes sense

File: src/modules/flow/iio/iio.json [hunk 1]
```diff
@@ -201,7 +201,7 @@
         "close": "temperature_close",
         "open": "temperature_open"
       },
-      "name": "iio/temperature",
+      "name": "iio/temperature-sensor",
```
Comment (edersondisouza) [inline src/modules/flow/iio/iio.json]: Anything against thermometer?

**Comment by edersondisouza** [comment]
looks ok

File: src/modules/flow/iio/iio.json (hunk 1 above)
Comment (glima) [inline src/modules/flow/iio/iio.json]: Me gusta. Might as well change the others.

**Comment by glima** [comment]
Merged

**Pull request closed without merging**